api/polyfills/browserAliases.json

```diff
@@ -1,54 +0,0 @@
-{
```

Comment:
This is implementation specific to uadetector library, so I've moved it into the service class instead of being a config. Without this, the service will not work properly.